promotion/src/main/java/org/cryptimeleon/incentive/promotion/vip/UpgradeVipZkpTokenUpdate.java

```diff
@@ -47,6 +47,7 @@ public UpgradeVipZkpTokenUpdate(Representation representation) {
      */
     public UpgradeVipZkpTokenUpdate(UUID rewardId, String rewardDescription, int toVipStatus, Integer accumulatedCost) {
         // TODO we could add the advantage of having that VIP level to the side effect
+        // Yes, replace Side Effect by that!
```

Comment:
Is this put into an issue?

Reply:
https://github.com/cryptimeleon/incentive-system/issues/96